README.md

```diff
@@ -1,18 +1,33 @@
 # Project-Protectors
-Feladataink:
-
-[ ] A repo a célnak megfelelően struktúrált.
-
-[ ] A repo minden szükséges fájlt tartalmaz, de csak a szükségeseket.
-
-[ ] A repo-t hatékonyan használják.
-
-[ ] A fejlesztést a verziókezelő eszközeivel megfelelően támogatják.
-
-[ ] A tervezett projekten való érdemi munka (azaz nem két hordónyi szemét commit!) található a projektben.
-
-Ami kell a projecthez:
-
-[ ] Egy Sprite osztály amiből majd leszármaznak a különböző karakter osztályok.
-
-[ ] A GameEngine osztáy kibővítése egy főmenüvel, egy küldetések menüvel, egy karakter választás menüvel és a harci képernyővel.
+Members: Zászlós Dorottya Beáta, Árvai Balázs, Bíró Benjámin, Fodor Ádám
+
+# Meetings
+Minden héten kedd 18:00-tól, illetve szobmat 12:00-től egy pár perces beszélgetésre kerül sor discord-on, mely során mindenki beszámol
+arról min dolgozott az elmúlt időszakban. Ha valaki nem tud részt venni valamelyik alkalmon chat-en illik előre pár mondatban leírni amit mondana.
+
+# Workflow
+A GitHub-on a feladatok a beadandók alapján projektekre vannak taggolva. A projektekhez tartozó részfeladatok tovább vannak bontva ticket-ekre.
+Ha egy ticket-en valaki dolgozni kezd, az alábbi lépéseket kövesse:
+    1. A ticket-et assign-olja magához
+    2. A ticket állapotát állítsa át In progress-re
+    3. Hozzon létre egy új branch-en a legfrissebb main-ről leágazva, melynek a nevében legyen benne a ticket neve és száma.
+    4. Ha késznek ítéli a feladatot akkor nyisson egy pull request-et a main branch-re.
+    5. A pr-hez adja hozzá a ticket-et, hogy a rendszer automatikusan tudja mozgatni a jegyet.
+    6. Ha a review-t végző fejlesztők módosításokat kérnek nem szükséges új pr létrehozása, csupán a branch-re fel kell pusholni az új commit-okat.
+    7. Ha legalább két másik fejlesztő approve-olta a review-t, akkor be kell mergelni és lezárni a branchet.
+    8. Ellenőrizni a jegy megfelelően lezárult (Done).
```

Comment:
Így nem lesz sortörés a bulletpointok között. Ha átváltasz erre a branchre, akkor látni fogod. Igaz ez a következő két bekezdésre is.